docs/dev_sliding_dot_product.ipynb

```diff
@@ -0,0 +1,981 @@
+{
```

Comment:
<p>Add the following lines to the code:</p><pre class="ql-syntax" spellcheck="false">n = 2 ** 15
m_values = range(10, 1000 + 10, 10)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/939/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>